cms/templates/widgets/header.html

```diff
@@ -229,6 +230,11 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Current
                   <li class="nav-item nav-account-dashboard">
                     <a href="/">${_("{studio_name} Home").format(studio_name=settings.STUDIO_SHORT_NAME)}</a>
                   </li>
+                  % if GlobalStaff().has_user(user):
```

Comment:
@cptvitamin @adampalay I had added this dropdown option to user-nav. Is this okay ?